src/serac/infrastructure/profiling.cpp

```diff
@@ -78,5 +78,34 @@ void setCaliperMetadata([[maybe_unused]] const std::string& name, [[maybe_unused
   cali_set_global_uint_byname(name.c_str(), data);
 #endif
 }
+
+void startCaliperRegion([[maybe_unused]] const char* name)
+{
+#ifdef SERAC_USE_CALIPER
+  CALI_MARK_BEGIN(name);
+#endif
+}
+
+void startCaliperRegion([[maybe_unused]] const std::string& name)
+{
+#ifdef SERAC_USE_CALIPER
+  CALI_MARK_BEGIN(name.c_str());
+#endif
+}
```

Comment:
I think that the `const char* -> std::string` conversion would make the overloading here unnecessary, is the intent to just avoid the extra work where possible?

Reply:
When I had tried passing in `std::string` earlier I had an issue, but I can test it again.